diff --git a/verl/utils/debug/performance.py b/verl/utils/debug/performance.py
--- a/verl/utils/debug/performance.py
+++ b/verl/utils/debug/performance.py
@@ -168,6 +168,6 @@ def reduce_timing(timing_raw: Dict[str, float]) -> Dict[str, float]:
         Dict[str, float]: Reduced timing information.
     """
-    if not dist.is_initialized():
+    if not dist.is_initialized() or dist.get_world_size() == 1:
         return timing_raw
 
     key_list, timing_list = [], []
